gamedata/tf2.utils.nosoop.txt

```diff
@@ -294,7 +294,7 @@
 				// unique xref to call containing unique xref string "add_maxhealth"
 				"library"		"server"
 				"linux"			"@_ZN15CTFPlayerShared18GetMaxBuffedHealthEbb"
-				"windows"		"\x55\x8B\xEC\x83\xEC\x14\x53\x8B\xD9\x56\x57\x8B\x8B\x2A\x01\x00\x00"
+				"windows"		"\x55\x8B\xEC\x83\xEC\x14\x53\x8B\xD9\x56\x57\x8B\x8B\x8C\x01\x00\x00"
```

Comment:
It looks like the `mov ecx, [ebx+18Ch]` operation is unchanged in the latest version &mdash; I usually wildcard the low byte of the offset since member offsets tend to be much more brittle than the rest of the code.